src/functionalTest/java/uk/gov/hmcts/reform/iacaseapi/util/StringResourceLoader.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.hmcts.reform.iacaseapi.util;
+
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.springframework.core.io.Resource;
+import org.springframework.core.io.support.PathMatchingResourcePatternResolver;
+import org.springframework.util.StreamUtils;
+
+public final class StringResourceLoader {
+
+    private StringResourceLoader() {
+        // noop
+    }
+
+    public static Map<String, String> load(String locationPattern) throws IOException {
+
+        Resource[] resources =
+            new PathMatchingResourcePatternResolver()
+                .getResources(locationPattern);
+
+        return
```

Comment:
Could just be my subjectivity but this stream seems hard to read

Reply:
Fair enough -- I will take the anonymous function and extract it into a method which should make it all read better 👍